CMakeLists.txt

```diff
@@ -71,6 +71,18 @@ endif()
 # include(sdk/cmake/bison-flex.cmake)
 
 if(NOT CMAKE_CROSSCOMPILING)
+
+    set(ENABLE_EXPORT_VERSIONING TRUE)
+    if (ENABLE_EXPORT_VERSIONING)
+        set(NTDLL_VISTA ntdll)
```

Comment:
IMHO we should find a better place for these than the top-level CMakeLists, especially if the list of these libraries is expected to grow.

Reply:
I don't see why they shouldn't be within the same folder as the main dll. They shoudlN#t need their own folder or CMakeList file.